guides/common/modules/proc_registering-hosts-by-using-the-bootstrap-script.adoc

```diff
@@ -17,6 +17,11 @@ To use the bootstrap script, you must install it on the host.
 As the script is only required once, and only for the `root` user, you can place it in `/root` or `/usr/local/sbin` and remove it after use.
 This procedure uses `/root`.
 
+.Limitations
+Reverse proxying on {SmartProxies} is disabled by default for security reasons.
+Therefore, the bootstrap script will not work if you register hosts through {SmartProxy}.
```

Comment:
```suggestion
Therefore, the bootstrap script does not work if you register hosts through {SmartProxy}.
```
Very small nit. RHSSG says to avoid future tense. Other than that, LGTM.